src/topics/news/sources-of-truth.md

```diff
@@ -10,7 +10,7 @@ author:
   git: mcalligator
   name: Angus
 date: 2024-04-30
-# linkedin: m-ld-io_news-activity-6975412303856812033--VEC
+https://www.linkedin.com/feed/update/urn:li:activity:7191426602377838592

```

Comment:
The `linkedin` YAML key is missing. Also, the link should only be the last path element of what looks like a different URL style... I'll look into it next (although I'm about to be on a bus, so even worse for doing anything...)

Reply:
Ah yes, I see: I naively pasted the link without regard to the required data structure; apologies for sounding the alarm too early.  I'll address that and give it another go.